### ISSUE TITLE
Itch 25.5.1 fails to run on Void linux

### ISSUE THREAD

**Issue by Foidii** [itchio/itch#2905]
I download a itch.io installer form website, install it. After instalation app tries to start, there apeares a white window, after wich itch crashes

Kernel: 6.1.8_1 

driver:
nvidia 525.85.05

**Comment by TheK0tYaRa** [comment]
Did you try checking required libraries with patchelf yet?

**Comment by Foidii** [comment]
I did, just now

**Comment by Foidii** [comment]
> I did, just now

libpthread.so.0
libgio-2.0.so.0
libgobject-2.0.so.0
libglib-2.0.so.0
libgtk-3.so.0
libgdk-3.so.0
libpangocairo-1.0.so.0
libpango-1.0.so.0
libcairo-gobject.so.2
libcairo.so.2
libgdk_pixbuf-2.0.so.0
libfontconfig.so.1
libc.so.6

**Comment by TheK0tYaRa** [comment]
`whereis` for each of them. The ones that don't provide a file path are missing in your system.

**Comment by Foidii** [comment]
I am missing the first one, but it aint in offical repo, or it is called differently

**Comment by Foidii** [comment]
wait, I misstyped. All of them are installed

**Comment by TheK0tYaRa** [comment]
When you run it in a terminal - does it output `GPU process isn't usable` ?

**Comment by Foidii** [comment]
itch or the libs? If u mean itche then it runs normaly untill the white window happens again

**Comment by TheK0tYaRa** [comment]
Indeed the window is white before SIGTRAP. But what is the output of it being ran in the terminal?

**Comment by Foidii** [comment]
2023/02/08 01:43:09 itch-setup will log to /tmp/itch-setup-log.txt
2023/02/08 01:43:09 =========================================
2023/02/08 01:43:09 itch-setup "v1.26.0, built on Apr 22 2021 @ 03:48:12, ref 48f97b3e7b0b065a2478811b8d0ebcae414845fd" starting up at "2023-02-08 01:43:09.623890371 +0100 CET m=+0.001848491" with arguments:
2023/02/08 01:43:09 "/home/adam/Downloads/itch-setup"
2023/02/08 01:43:09 =========================================
2023/02/08 01:43:09 App name specified at build time: itch
2023/02/08 01:43:09 Locale:  en-US
2023/02/08 01:43:09 Initializing installer GUI...
2023/02/08 01:43:09 Using GTK UI
2023/02/08 01:43:09 Initializing (itch) multiverse @ (/home/adam/.itch)
2023/02/08 01:43:09 (/home/adam/.itch)(current = "25.5.1", ready = "")
2023/02/08 01:43:09 Loading image resources...
2023/02/08 01:43:09 Setting up progress bar...
2023/02/08 01:43:09 Positioning and showing window...
2023/02/08 01:43:09 Will install version 25.5.1
2023/02/08 01:43:10 Reading signature...
2023/02/08 01:43:10 Installing 197.43 MiB (295 files, 25 dirs, 0 symlinks)
2023/02/08 01:43:10 Looks like (25.5.1) is already installed to (/home/adam/.itch/app-25.5.1)
2023/02/08 01:43:10 Let's just heal that
2023/02/08 01:43:10 Installing to (/home/adam/.itch/app-25.5.1)
2023/02/08 01:43:10 Healing (/home/adam/.itch/app-25.5.1)...
2023/02/08 01:43:12 0 B was healed @ 0 B/s (a few seconds total)
2023/02/08 01:43:12 Healed in-place
2023/02/08 01:43:12 No validate handler, assuming good!
2023/02/08 01:43:12 Determining whether or not we've been installed via an OS package...
2023/02/08 01:43:12 Copying self to (/home/adam/.itch/itch-setup)
2023/02/08 01:43:12 install (/home/adam/.itch/itch)
2023/02/08 01:43:12 install (/home/adam/.itch/icon.png)
2023/02/08 01:43:12 install (/home/adam/.local/share/applications/io.itch.itch.desktop)
2023/02/08 01:43:12 Updating desktop database for (/home/adam/.local/share/applications)
Search path is now: [/home/adam/.local/share/applications]
File "/home/adam/.local/share/applications/Tabletop Simulator.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/Dota 2.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/Deep Rock Galactic.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Fallout - New Vegas [GOG.com]/Tools/G.E.C.K..desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Fallout - New Vegas [GOG.com]/Uninstall Fallout - New Vegas.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Fallout - New Vegas [GOG.com]/Fallout - New Vegas.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Genshin Impact/Genshin Impact.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Genshin Impact/Uninstall Genshin Impact.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Radmin VPN/Radmin Viewer 3.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Radmin VPN/Radmin VPN.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Itch Corp/itch.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Riot Games/League of Legends.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Riot Games/Riot Client.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/The Elder Scrolls II - Daggerfall/Tools/Graphic Mode Setup.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/The Elder Scrolls II - Daggerfall/Documents/Manual.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/The Elder Scrolls II - Daggerfall/Documents/Reference Card.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/The Elder Scrolls II - Daggerfall/Uninstall The Elder Scrolls II - Daggerfall.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/The Elder Scrolls II - Daggerfall/The Elder Scrolls II - Daggerfall.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/Sacrifice/Uninstall Sacrifice.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/Sacrifice/Sacrifice.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/GOG.com/Sacrifice/Documents/Manual.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Succubus [GOG.com]/Succubus DirectX 11.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Succubus [GOG.com]/Uninstall Succubus.desktop" lacks MimeType key
File "/home/adam/.local/share/applications/wine/Programs/Succubus [GOG.com]/Succubus.desktop" lacks MimeType key
2023/02/08 01:43:12 Launching (25.5.1) from (/home/adam/.itch/app-25.5.1)
2023/02/08 01:43:12 Kernel should support SUID sandboxing, leaving it enabled
2023/02/08 01:43:12 App launched, getting out of the way

**Comment by TheK0tYaRa** [comment]
That is the installer. It seems to work fine. What the `itch` binary outputs? The `~/.itch/itch` one

**Comment by Foidii** [comment]
itch-setup will log to /tmp/itch-setup-log.txt
2023/02/08 01:49:09 =========================================
2023/02/08 01:49:09 itch-setup "v1.26.0, built on Apr 22 2021 @ 03:48:12, ref 48f97b3e7b0b065a2478811b8d0ebcae414845fd" starting up at "2023-02-08 01:49:09.009017754 +0100 CET m=+0.002219132" with arguments:
2023/02/08 01:49:09 "/home/adam/.itch/itch-setup"
2023/02/08 01:49:09 "--prefer-launch"
2023/02/08 01:49:09 "--appname"
2023/02/08 01:49:09 "itch"
2023/02/08 01:49:09 "--"
2023/02/08 01:49:09 =========================================
2023/02/08 01:49:09 App name specified on command-line: itch
2023/02/08 01:49:09 Locale:  en-US
2023/02/08 01:49:09 Initializing installer GUI...
2023/02/08 01:49:09 Using GTK UI
2023/02/08 01:49:09 Initializing (itch) multiverse @ (/home/adam/.itch)
2023/02/08 01:49:09 (/home/adam/.itch)(current = "25.5.1", ready = "")
2023/02/08 01:49:09 Launch preferred, attempting...
2023/02/08 01:49:09 Launching (25.5.1) from (/home/adam/.itch/app-25.5.1)
2023/02/08 01:49:09 Kernel should support SUID sandboxing, leaving it enabled
2023/02/08 01:49:09 App launched, getting out of the way

Then white window apears for a moment and after it dies

**Comment by TheK0tYaRa** [comment]
Are you sure that you are running exactly `itch` binary?

**Comment by Foidii** [comment]
yes

**Comment by TheK0tYaRa** [comment]
Does `.itch` directory contain `chrome_100_percent.pak  chrome_200_percent.pak  icudtl.dat  itch  libffmpeg.so  libvk_swiftshader.so  locales/  resources/  resources.pak  snapshot_blob.bin  v8_context_snapshot.bin  version  vk_swiftshader_icd.json` ?

**Comment by Foidii** [comment]
these files are in /.itch/app-25.5.1/

**Comment by TheK0tYaRa** [comment]
Try running `~/.itch/app-25.5.1/itch`

**Comment by Foidii** [comment]
INFO itch@25.5.1 on electron@10.4.5 in production
00:55:17.055 DEBUG (m/reac/preb/load-preferences) imported preferences: {"downloadSelfUpdates":true,"offlineMode":false,"installLocations":{},"defaultInstallLocation":"appdata","isolateApps":false,"closeToTray":true,"readyNotification":true,"showAdvanced":false,"openAtLogin":false,"openAsHidden":false,"manualGameUpdates":false,"preventDisplaySleep":true,"preferOptimizedPatches":false,"disableBrowser":false,"enableTabs":false,"lastSuccessfulSetupVersion":"25.5.1","importedOldInstallLocations":true,"gotMinimizeNotification":true}
00:55:17.105 DEBUG (m/reac/preb/load-preferences) imported preferences: {"downloadSelfUpdates":true,"offlineMode":false,"installLocations":{},"defaultInstallLocation":"appdata","isolateApps":false,"closeToTray":true,"readyNotification":true,"showAdvanced":false,"openAtLogin":false,"openAsHidden":false,"manualGameUpdates":false,"preventDisplaySleep":true,"preferOptimizedPatches":false,"disableBrowser":false,"enableTabs":false,"lastSuccessfulSetupVersion":"25.5.1","importedOldInstallLocations":true,"gotMinimizeNotification":true}
00:55:17.109 DEBUG (m/net/register-itch-protocol) Registering itch: protocol for session with user agent Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) itch/25.5.1 Chrome/85.0.4183.121 Electron/10.4.5 Safari/537.36
00:55:17.142 INFO (m/reac/preboot) preboot ran in 37.00ms
00:55:17.143 DEBUG (m/reac/open-at-login) Updating login item settings, open=false, hidden=false
00:55:17.143 DEBUG (m/reac/open-at-login) Copying /usr/share/applications/io.itch.itch.desktop => /home/adam/.config/autostart/io.itch.itch.desktop
00:55:17.144 DEBUG (m/reac/winds) Waiting on root window to dispatch boot
[4847:0208/015517.453567:FATAL:gpu_data_manager_impl_private.cc(442)] GPU process isn't usable. Goodbye.

**Comment by TheK0tYaRa** [comment]
`GPU process isn't usable` indeed

**Comment by Foidii** [comment]
How do I fix it?

**Comment by TheK0tYaRa** [comment]
Well, i've done my part for the devs to try help us both. I have the same issue on Gentoo after updating Mesa. Maybe it's what caused the issue. On version 23.0.0 currently.

**Comment by Foidii** [comment]
so just wait for a fix?

**Comment by TheK0tYaRa** [comment]
I will try reverting my mesa upgrade. Waiting is always an option, but i prefer acting.

**Comment by bgvaughan** [comment]
After some web searches on that last error message, it sounds like it's an issue with a recent version of electron, running on Nvidia. A workaround suggested was launching the app with "--no-sandbox", and that does seem to work.

~/.itch/itch --no-sandbox

**Comment by leafo** [comment]
Closing out old tickets for very old versions of the app. Please update to 26.9.0 or above and create a new issue if you are still having problems.
